### PR TITLE
Add tests for acspRegistrationService

### PR DESCRIPTION
**Pull request by swalke-ch**
Add tests for acspRegistrationService

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/test/src/services/acspRegistrationService.test.ts [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+                    data: {
+                        id: "example@example.com",
+                        typeOfBusiness: "LIMITED"
+                    }
+                }
+            });
+
+            const acspResponse: AcspResponse = await postAcspRegistration(session, TRANSACTION_ID, acsp);
+
+            console.log(acspResponse);
```
Comment (sardhani-ch) [inline src/test/src/services/acspRegistrationService.test.ts:56]: Can you remove the console.log

File: src/test/src/services/acspRegistrationService.test.ts [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+            await expect(getAcspRegistration(session, TRANSACTION_ID, EMAIL_ID)).rejects.toEqual({ httpStatusCode: StatusCodes.NOT_FOUND });
+        });
+
+        it("Should throw an error when acsp api returns no resource", async () => {
+            mockGetAcspRegistration.mockResolvedValueOnce({
+                httpStatusCode: StatusCodes.OK
+            });
+
+            await expect(getAcspRegistration(session, TRANSACTION_ID, EMAIL_ID)).rejects.toEqual({ httpStatusCode: StatusCodes.OK });
```
Comment (sardhani-ch) [inline src/test/src/services/acspRegistrationService.test.ts:134]: want to understand that for no resource, status code 200/201 is correct?

File: src/test/src/services/acspRegistrationService.test.ts (hunk 1 above)
Comment (swalke-ch) [reply src/test/src/services/acspRegistrationService.test.ts:56]: removed now

File: src/test/src/services/acspRegistrationService.test.ts (hunk 2 above)
Comment (swalke-ch) [reply src/test/src/services/acspRegistrationService.test.ts:134]: updated the httpresponse to return INTERNAL_SERVER_ERROR

**Review by sardhani-ch — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**